src/test/regress/expected/ensure_no_shared_connection_leak.out

```diff
@@ -117,6 +117,13 @@ SELECT pg_reload_conf();
 
 \c - - - :master_port
 SET search_path TO ensure_no_shared_connection_leak;
+-- invalidate inactive shared connections
```

Comment:
minor: `invalidate inactive shared connections` -> `invalidate inactive shared connections to ensure  that test output doesn't change due to inactive nodes`

Reply:
As I removed this udf, this is no more relevant.